### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by menyagah**
# In this section we:
 - Refactored the code into classes
 - Fixed linter errors
 - Fixed a bug with input fields not clearing upon submission

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,94 +1,84 @@
 /* eslint no-unused-vars: 0 no-undef: 0 */
+let id = 0;
 const books = [];
```
Comment (mimipeshy) [inline main.js]: - [ ] From the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes.md)  first point, it's okay to use  `arrays` in this milestone but it has to be done inside classes eg 
```
class Books {
  constructor() {
    this.books = [];
  }
  getBooks() {
    return this.books;
  }
}
```
Kindly fix this

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @Team 👋🏾 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
#### Highlights

- you have maintained the app functionality ✔️ 
- good GitHub workflow ✔️ 
- not forming any linter errors ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Comment by lily-coder** [comment]
We have implemented all changes as instructed by the reviewer.

**Review by mimipeshy — APPROVED** [review]
## Hi @Team 👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**